captain-definition

```diff
@@ -5,10 +5,14 @@
                 "COPY ./ /app",
                 "WORKDIR /app",
                 "RUN apt-get update",
-                "RUN apt-get install -y build-essential libpq-dev",
+                "RUN apt-get install -y build-essential libpq-dev postgresql-client cron nano gettext librsync-dev gnupg-agent",
```

Comment:
what is gettext needed for?

Reply:
gettext is for solving the error:
```
    sh: 1: msgfmt: not found
```

Could only make the installation to complete if i place them in the captain-definition rather than the requirements.txt